server/tests/ballot_comparison/snapshots/snap_test_cvrs.py

```diff
@@ -7,1990 +7,7 @@
-        "tabulator": "TABULATOR2",
-    },
-    {
-        "ballot_position": 2,
-        "batch_name": "BATCH3",
-        "imprinted_id": "unique-identifier-08",
-        "interpretations": "1,0,0,0,1,0",
-        "tabulator": "TABULATOR2",
-    },
-    {
-        "ballot_position": 3,
-        "batch_name": "BATCH3",
-        "imprinted_id": "unique-identifier-09",
-        "interpretations": "1,0,0,1,0,0",
-        "tabulator": "TABULATOR2",
-    },
-    {
-        "ballot_position": 1,
-        "batch_name": "BATCH4",
-        "imprinted_id": "unique-identifier-10",
-        "interpretations": "1,0,0,0,1,0",
-        "tabulator": "TABULATOR2",
-    },
-    {
-        "ballot_position": 2,
-        "batch_name": "BATCH4",
-        "imprinted_id": "unique-identifier-11",
-        "interpretations": "1,1,1,1,1,0",
-        "tabulator": "TABULATOR2",
-    },
-    {
-        "ballot_position": 3,
-        "batch_name": "BATCH4",
-        "imprinted_id": "unique-identifier-12",
-        "interpretations": ",,1,0,0,0",
-        "tabulator": "TABULATOR2",
-    },
-    {
-        "ballot_position": 4,
-        "batch_name": "BATCH4",
-        "imprinted_id": "unique-identifier-13",
-        "interpretations": ",,1,0,0,0",
-        "tabulator": "TABULATOR2",
-    },
-    {
-        "ballot_position": 5,
-        "batch_name": "BATCH4",
-        "imprinted_id": "unique-identifier-14",
-        "interpretations": ",,0,0,0,1",
-        "tabulator": "TABULATOR2",
-    },
-]
-
-snapshots["test_hart_cvr_upload_with_scanned_ballot_information 10"] = {
-    "Contest 1": {
-        "choices": {
-            "Choice 1-1": {"column": 0, "num_votes": 7},
-            "Choice 1-2": {"column": 1, "num_votes": 3},
-        },
-        "total_ballots_cast": 11,
-        "votes_allowed": 1,
-    },
-    "Contest 2": {
-        "choices": {
-            "Choice 2-1": {"column": 2, "num_votes": 6},
-            "Choice 2-2": {"column": 3, "num_votes": 3},
-            "Choice 2-3": {"column": 4, "num_votes": 3},
-            "Write-In": {"column": 5, "num_votes": 1},
-        },
-        "total_ballots_cast": 14,
-        "votes_allowed": 1,
-    },
-}
-
-snapshots["test_hart_cvr_upload_with_scanned_ballot_information 2"] = {
-    "Contest 1": {
-        "choices": {
-            "Choice 1-1": {"column": 0, "num_votes": 7},
-            "Choice 1-2": {"column": 1, "num_votes": 3},
-        },
-        "total_ballots_cast": 11,
-        "votes_allowed": 1,
-    },
-    "Contest 2": {
-        "choices": {
-            "Choice 2-1": {"column": 2, "num_votes": 6},
-            "Choice 2-2": {"column": 3, "num_votes": 3},
-            "Choice 2-3": {"column": 4, "num_votes": 3},
-            "Write-In": {"column": 5, "num_votes": 1},
-        },
-        "total_ballots_cast": 14,
-        "votes_allowed": 1,
-    },
-}
-
-snapshots["test_hart_cvr_upload_with_scanned_ballot_information 3"] = [
+snapshots["test_hart_cvr_upload_with_duplicate_batch_names 1"] = [
```

Comment:
I don't really understand why anything in this file changed but I think the snapshots were just out of date ?